crates/router/src/core/admin.rs

```diff
@@ -387,6 +420,9 @@ pub async fn update_payment_connector(
         pm_enabled
             .iter()
             .flat_map(|payment_method| {
+                validate_pm_enabled(payment_method)
+                    .change_context(errors::ParsingError)
```

Comment:
Shouldn't this be `ValidationError`

Reply:
anyway we can throw the validation error inside the current iteration